var/spack/repos/builtin/packages/py-tatsu/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTatsu(PythonPackage):
+    """竜 TatSu (the successor to Grako) is a tool that takes grammars in
+    a variation of EBNF as input, and outputs memoizing (Packrat) PEG
+    parsers in Python."""
+
+    homepage = "https://github.com/neogeny/tatsu"
+    url      = "https://pypi.io/packages/source/T/TatSu/TatSu-4.4.0.zip"
+
+    version('4.4.0', sha256='80713413473a009f2081148d0f494884cabaf9d6866b71f2a68a92b6442f343d')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-colorama@0.4:', type=('build', 'run'))
+    depends_on('py-dataclasses@0.6:', type=('build', 'run'), when='^python@3.6.0:3.6.99')
+    depends_on('py-mypy@0.641:', type=('build', 'run'))
+    depends_on('py-pyyaml@3.12:', type=('build', 'run'))
+    depends_on('py-regex@2018.8:', type=('build', 'run'))
+    depends_on('py-typing@3.6:', type=('build', 'run'), when='^python@:3.4.99')
```

Comment:
Where did you find these requirements? They aren't in the `setup.py`.

Reply:
https://github.com/neogeny/TatSu/blob/master/requirements.txt

Reply:
I'm now really unsure on which we should follow... `requirements.txt` seems to be what `pip` is looking for, but it seems to be used in addition to what is state in `setup.py` and/or `setup.cfg`...

Reply:
Only trust `setup.py` and `setup.cfg` in the current release tarball, never trust `requirements.txt` on master/develop.